src/lib.c

```diff
@@ -0,0 +1 @@
+#include "../include/despacer.h"
```

Comment:
What is your expectation that this would do?

Reply:
My expectation is that it would export no useful symbol. Effectively, it should be a useless binary.

Reply:
That might be the issue. Using the method I am using, D needs an object file or a library file so it can link into. That's why I added a C file. I need to ask to see if it is possible to link header only libraries.

Reply:
@lemire It turns out that it is not possible to use this library from other languages unless we make it a normal library. I did this in the commits that I pushed.